notebooks/Data_Attribute_Recommendation_Generic_Model_Template.ipynb

```diff
@@ -0,0 +1,1524 @@
+        "*Boosters are only available on the Trial landscape. If you are using a production environment, please follow this tutorial to manually [create a service instance and a service key](https://developers.sap.com/tutorials/cp-aibus-dar-service-instance.html)*."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "cjQ0_esrKpfd"
+      },
+      "source": [
+        "In the Boosters tab, enter \"Data Attribute Recommendation\" into the search box. Then, select the\n",
+        "service tile from the search results: \n",
+        "    \n",
+        "![trial_locate_dar_booster.png](images/trial_locate_dar_booster.png)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "0yGm9c4dKpfe"
+      },
+      "source": [
+        "The resulting screen shows details of the booster pack. Here, click the \"Start\" button and wait a few seconds.\n",
+        "\n",
+        "![trial_start_booster.png](images/trial_start_booster.png)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "5rL_rFntKpfe"
+      },
+      "source": [
+        "Once the booster is finished, click the \"go to Service Key\" link to obtain your service key.\n",
+        "\n",
+        "![trial_booster_finished.png](images/trial_booster_finished.png)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "LOMd19zhKpfe"
+      },
+      "source": [
+        "Finally, download the key and save it to disk.\n",
+        "\n",
+        "![trial_download_key.png](images/trial_download_key.png)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "bkk5nSzYKpff"
+      },
+      "source": [
+        "## Loading the service key into your Jupyter Notebook"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "2gjYa-qnKpff"
+      },
+      "source": [
+        "Once you downloaded the service key from the Cockpit, upload it to your notebook environment. The service key must be uploaded to same directory where the `Data_Attribute_Recommendation_*_Model_Template.ipynb` file is stored.\n",
+        "\n",
+        "When using Jupyterlab, a file browser is visible to the left of the notebook view. Click the upload button here to upload the `default_key.json` file we downloaded earlier from the BTP Cockpit.\n",
+        "\n",
+        "\n",
+        "![jupyterlab_upload_button.png](images/service_key_main_jupyter_page.png)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "Once you click the upload button, a file chooser dialog will open where you can select the `default_key.json`:\n",
+        "After the upload finished successfully, you should see the `default_key.json` in the file browser.\n",
+        "\n",
+        "![jupyterlab_service_key_uploaded_successfully.png](images/service_key_upload.png)\n",
```

Comment:
The screenshots with the jupyter page and the notebook are still a bit outdated ("teched2020", and "key.json" are showing up there) - could we also update these? Btw, I find it quite nice to have the images as paths instead of attachments - they are still displayed in the browser (if you click on the notebook in github)... They are even visible in something like [mybinder.org](https://hub.gke2.mybinder.org/user/sap-data-attrib-tion-python-sdk-04i85m7y/lab/tree/notebooks/Data_Attribute_Recommendation_Generic_Model_Template.ipynb); with that, we can also share those same screenshots for the setup between the notebooks :)

Reply:
Thanks! Yes, I am also able to view the screenshots from the notebook on the branch. I've updated the branch to remove this screenshot as it looks like the text explains what is required and the screenshot was misleading. Hope this looks better.